cmake/modules/RootMacros.cmake

```diff
@@ -282,6 +282,95 @@ function(ROOT_GENERATE_DICTIONARY dictionary)
     set(CMAKE_INSTALL_LIBDIR ${CMAKE_CURRENT_BINARY_DIR})
     set(libprefix "")
   endif()
+  
+  
+  if((CMAKE_PROJECT_NAME STREQUAL ROOT) AND (TARGET ${ARG_MODULE}))
+    set(incdirs)
+
+    # for a time been, roocling makes strange error
```

Comment:
Could you clarify in the comment what is the error and what is the context

Reply:
Yes, I will do it. I will just copy your comments from other place:
```
      # FIXME: We prepend ROOTSYS/include because if we have built a module
      # and try to resolve the 'same' header from a different location we will
      # get a redefinition error.
      # We should remove these lines when the fallback include is removed. Then
      # we will need a module.modulemap file per `inc` directory.
```